jobs/dynatrace-oneagent/templates/pre-start.erb

```diff
@@ -67,6 +68,17 @@ if [[ "$DOWNLOADURL" == "" ]]; then
     DOWNLOADURL="$API_URL/v1/deployment/installer/agent/unix/default/latest?Api-Token=$API_TOKEN"
 fi
 
+# set validatedownload to false (fallback) if not set
+if [[ "$VALIDATEDOWNLOAD" == "" ]]; then
+    installLog "Disabling download validation"
+    VALIDATEDOWNLOAD=false    
+fi
+
+if [[ "$VALIDATEDOWNLOAD" == true ]]; then
+    installLog "Enabling download validation"
+    VALIDATEDOWNLOAD=true
+fi
```

Comment:
The variable `VALIDATEDOWNLOAD` is undefined for any other value, or well, it's set to whatever came from the properties. I suggest an if else.